### PR TITLE
update release note as tizen-8-0-m2.md

### PR DESCRIPTION
**Pull request by seokwon-jung**
### Change Description ###

Describe your changes here.
update release note as tizen-8-0-m2.md
For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by safir-srbd — COMMENTED** [review]
@seokwon-jung Thank you for making this PR.

It seems that the file was not added to `toc_all.md` and the LNB is not appearing. Please add the `tizen-8-0-m2.md` to the `toc_all.md` file.

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+#### Fixes
+
+- Various partial update defects have been fixed.
+- A transform matrix calculation defect has been fixed.
+- A screen rotation defect has been fixed.
+- Various BMP decoder defects have been fixed.
+- Various text defects have been fixed.
+- Various Aurum defects have been fixed.
+- Various touch and gesture defects have been fixed.
+- Fixs the crash issue for web application
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:281]:
```suggestion
- Fixes the crash issue for web application
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+  - Improves the performance of finding element
+  - Supports for window rotation function
+  - Improves the performance of creating xPath
+- UI Analyzer(internal Tool)
+  - Supports for the information of rotated window
+  - Supports for enhanced test recording
+  - Supports for testing validataions
+  - Supports for playing the recorded tests and showing the result
+  - Supports for saving the recorded tests and loading the saved file
+  - Supports for playing the smoke tests automatically
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:268]:
```suggestion
  - Support for enabling xPath command
  - Support for getting the window rotation angle
  - Support for getting information whether the application is idle or not
  - Improves the performance of finding element
  - Support for window rotation function
  - Improves the performance of creating xPath
- UI Analyzer(internal Tool)
  - Support for the information of rotated window
  - Support for enhanced test recording
  - Support for testing validation
  - Support for playing the recorded tests and showing the result
  - Support for saving the recorded tests and loading the saved file
  - Support for playing the smoke tests automatically
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+  - Some display functions have been changed to return not-supported error on a headless binary.
+- Media Content
+  - Deprecated some storage related APIs and unused media metadata have been removed.
+  - Deprecated thumbnail util APIs have been removed.
+- Audio Framework
+  - New APIs for sound effects have been added.
+    - Support Acoustic Echo Cancellation functionality.
+    - Support Noise Suppression based on neural network.
+    - Support Automatic Gain Control.
+    - Support to be able to get a reference playback stream when recording sound.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:314]:
```suggestion
    - Supports Acoustic Echo Cancellation functionality.
    - Supports Noise Suppression based on neural network.
    - Supports Automatic Gain Control.
    - Supports to be able to get a reference playback stream when recording sound.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+- MediaController
+  - Add new Capability APIs
+    - New C# APIs for PlaybackPosition, PlaylistCapability, CustomCommandCapability, SearchCapability has been added.
+
+### Network and connectivity
+
+#### New and changed features
+
+- Network
+  - Support 6GHz band for Wi-Fi 6E has been added.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:326]:
```suggestion
  - 6GHz band for Wi-Fi 6E has been added.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 5]
```diff
@@ -0,0 +1,507 @@
...
+  - New API for getting 6Ghz band support has been added.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:327]:
```suggestion
  - New API for getting 6GHz band support has been added.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 6]
```diff
@@ -0,0 +1,507 @@
...
+### Security
+
+#### New and changed features
+
+- Privacy privilege manager
+  - Privacy privilege manager (PPM) feature has been deprecated.
+- Key manager
+  - supports multi-staged encryption.
+  - supports importing and exporting wrapped key.
+  - supports key derivation with ECDH and KBKDF.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:358]:
```suggestion
  - Supports multi-staged encryption.
  - Supports importing and exporting wrapped key.
  - Supports key derivation with ECDH and KBKDF.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 7]
```diff
@@ -0,0 +1,507 @@
...
+
+### Web framework
+
+#### New and changed features
+
+- Web Engine
+  - SoundFocusManager for web media has been supported.
+  - Accessibility has been supported.
+  - H.264 codec for public profile has been supported.
+  - ewk_view_ime_window_set to support IME for NUIhas been added.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:388]:
```suggestion
  - ewk_view_ime_window_set to support IME for NUI has been added.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 8]
```diff
@@ -0,0 +1,507 @@
...
+- Maplike declaration has been added.
+- JavaScript Engine
+- ECMAscript feature (from ES2022) has been added.
+
+#### Fixes
+
+- Lightweight Web Engine
+- Network loader defects have been fixed.
+- Memory leak defects have been fixed.
+- Url parsing defects have been fixed.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:417]:
```suggestion
- URL parsing defects have been fixed.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 9]
```diff
@@ -0,0 +1,507 @@
...
+### Machine learning
+
+#### New and changed features
+
+- Machine Learning (ML) API updates
+  - ML.Service API Updates
+    - Add new APIs to separately manage ML model files. These APIs allow ML applications to exploit the latest ML model deployed without code-level modifications that require re-packaging and re-distribution of the whole application.
+    - AI Inference offloading between Tizen and TizenRT has been improved. The binary size of `NNStreamer-Edge` is optimized.
+    - Add new APIs for machine learning resource which contains images, audio, video and binary files. These APIs provides a method to share the data files those can be used for training or inferencing the AI models.
+    - Add new utility functions to handle the key-value style information. This informantion object can be used to update the configuration and check the internal status.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:472]:
```suggestion
    - Add new utility functions to handle the key-value style information. This information object can be used to update the configuration and check the internal status.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 10]
```diff
@@ -0,0 +1,507 @@
...
+  - ML.Inference API Updates
+    - Add new API to use extended rank limit. To maintain the backward compatibility, the default rank limit is 4. The extended rank is only used when new API is called.
+    - Support TensorFlow Lite v2.11.0 by default.
+  - ML.Training API Updates
+    - Add Identify Layer as a utility layer which flows everthing as it is.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:477]:
```suggestion
    - Add Identify Layer as a utility layer which flows everything as it is.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 11]
```diff
@@ -0,0 +1,507 @@
...
+  - Support for the model storage of Device MLOps has been added.
+    - Model management feature such as register, fetch active model, update, and delete has been added. Using this feature, ML applications can share their model with other applications.
+  - Pipeline based data repository and training feature has been added.
+    - New NNStreamer elements such as `datareposrc`, `datareposink`, and `tensor_trainer` have been added.
+    - Support for the training in AI pipeline has been added.
+- NNTrainer updates
+  - Support proactive swap which utilizes secondary storage for less memory consumption.
+    - Add Cache Pool/Cache Loader/Cache Element classes.
+    - Update Memory pool and planner for better utilization.
+  - Added Execution order and memory usage tarcing for debugging.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:496]:
```suggestion
  - Added Execution order and memory usage tracing for debugging.
```

**Review by safir-srbd — COMMENTED** [review]
@seokwon-jung Thank you for posting the Release Note for Tizen 8.0 M2.

I have reviewed the PR and found a few issues. Please implement them.

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 12]
```diff
@@ -0,0 +1,507 @@
...
+
+#### New and changed features
+
+- Kernel
+  - Kernel for Raspberry Pi 4 has been upgraded to version 5.15.92.
+  - CONFIG_PSI feature of Kernel is used to monitor resources.
+- System and resource management
+  - Provide the resource-manager/system-plugin-API interface to separate deviced-plugin from device management service.
+    - The system-plugin-backend-deviced-headed/headless git/package have been added to support the vendor specific code without any changes of device management core.
+  - Separate resourced-plugin including LMK(Low Memory Killer) & CPU boosting stall governors from resource management service.
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:28]:
```suggestion
  - Separate resourced-plugin including LMK (Low Memory Killer) & CPU boosting stall governors from resource management service.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 13]
```diff
@@ -0,0 +1,507 @@
...
+
+
+### Application framework
+
+#### New and changed features
+
+- Tizen Database Access Object (TDAO)
+  - New feature has been ready to access a database.
+    - Provide classes for DAO and Entity
+    - Provides Compile-time Verification of SQL queries
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:102]:
```suggestion
    - Provide classes for DAO and Entity.
    - Provides Compile-time Verification of SQL queries.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 14]
```diff
@@ -0,0 +1,507 @@
...
+
+### System (base)
+
+#### New and changed features
+
+- Open source
+  - python3-protobuf
+    - Version 3.9.2 has been upgraded to version 4.24.1.
+  - ncruses
+    - Version 6.2 has been upgraded to version 6.4
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:60]:
```suggestion
    - Version 6.2 has been upgraded to version 6.4.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 15]
```diff
@@ -0,0 +1,507 @@
...
+- cryptsetup
+  - Version 2.3.7 has been upgraded to version 2.6.1.
+
+#### Fixes
+
+- dbus
+  - CVE-2023-34969 patch has been applied.
+- btrfs
+  - CVE-2023-1611 patch has been applied.
+- systemd / cryptsetup / crash-worker / initrd / libdbuspolicy / upgrade
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:48]:
```suggestion
- systemd/cryptsetup/crash-worker/initrd/libdbuspolicy/upgrade
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 16]
```diff
@@ -0,0 +1,507 @@
...
+  - The IPC interface between the voice control engine and voice control clients has been changed from D-Bus to TIDL, which is a proper IPC method between applications.
+  - The C API to reduce background volume when the voice manager client wants has been added.
+- MMI Framework
+  - New MMI architecture has been applied.
+  - Supports workflows of voice touch, wakeupless command, and user recognition.
+- Gesture Framework
+  - Gesture framework has been deprecated.
+- Text Input
+  - The performance to get surrounding text has been improved.
+  - Autofill and Inputmethod setting application has been reimplemented based on NUI gadget.
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:192]:
```suggestion
  - Autofill and Input method setting application has been reimplemented based on NUI gadget.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 17]
```diff
@@ -0,0 +1,507 @@
...
+- Scene3D
+  - A new default camera for 3D scene has been added.
+  - Support for asynchronous loading of Model and SceneView has been added.
+  - Support for KHR_materials_specular and KHR_materials_ior extension of glTF has been added.
+  - Support for embedded texture data of glTF has been added.
+  - Cache manager for 3D models has been added.
+  - Support for glb format has been added.
+  - Support for equirectangular projection has been added.
+  - Support for NavigationMesh and PathFinding has been added.
+  - Support to modify Physically Based Rendering(PBR) material in runtime.
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:219]:
```suggestion
  - Support to modify Physically Based Rendering (PBR) material in runtime.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 18]
```diff
@@ -0,0 +1,507 @@
...
+  - Support for PlaceHolder.
+  - Support for image transition.
+  - Masking for runtime generated image.
+- Text
+  - Some text geometry APIs have been added.
+  - Some text span APIs have been added.
+  - Support for cache of font list
+- Drag and Drop
+  - Support for multiple windows on a single process has been added.
+- Performance / Memory Improvement
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:246]:
```suggestion
- Performance/Memory Improvement
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 19]
```diff
@@ -0,0 +1,507 @@
...
+- Aurum
+  - Support for UI context changed event has been added.
+  - Support for UI scrolling finished event has been added.
+  - Supports for enabling xPath command
+  - Supports for getting the window rotation angle
+  - Supports for getting information whether application idle or not
+  - Improves the performance of finding element
+  - Supports for window rotation function
+  - Improves the performance of creating xPath
+- UI Analyzer(internal Tool)
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:262]:
```suggestion
- UI Analyzer (internal Tool)
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 20]
```diff
@@ -0,0 +1,507 @@
...
+- Various partial update defects have been fixed.
+- A transform matrix calculation defect has been fixed.
+- A screen rotation defect has been fixed.
+- Various BMP decoder defects have been fixed.
+- Various text defects have been fixed.
+- Various Aurum defects have been fixed.
+- Various touch and gesture defects have been fixed.
+- Fixs the crash issue for web application
+- Fix the crash issue when click function works
+- Fix the memory leak
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:283]:
```suggestion
- Fix the crash issue when the click function works.
- Fix the memory leak.
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 21]
```diff
@@ -0,0 +1,507 @@
...
+  - The wheel event in touchpad has been supported.
+  - The server protocol implementation of single-pixel-buffer-v1 interface has been included.
+  - The server protocol implementation of wtz-blender interface has been included.
+  - The server protocol implementation of tws_service_kvm interface has been included.
+  - The relative pointer has been supported.
+  - The pointer lock has been supported.
+  - The input thread for processing key event has been separated for removing key event delivery delay element (Input Device check, Compositing, Capture, and so on).
+- Tizen WS Shell
+  - The KVM service has been added.
+    - KVM service provides Copy&Paste and Drag&Drop among multi-devices
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:155]:
```suggestion
    - KVM service provides Copy & Paste and Drag & Drop among multi-devices
```

File: docs/platform/release-notes/tizen-8-0-m2.md [hunk 22]
```diff
@@ -0,0 +1,507 @@
...
+  - The version of glslang has been upgraded to 1.3.239.
+- EOM
+  - EOM APIs have been deprecated.
+- Libinput
+  - The version of Libinput has been upgraded to 1.22.0.
+- Libxkbcommon
+  - The version of Libxkbcommon has been upgraded to 1.5.1.
+- TTS Framework
+  - Supports root daemon as TTS client.
+  - Supports on-demand reconnection of clients when TTS service abnormally terminated. This is useful for low-end devices when its cpu consumption is very high.
```
Comment (omar-srbd) [inline docs/platform/release-notes/tizen-8-0-m2.md:176]:
```suggestion
  - Supports on-demand reconnection of clients when TTS service is abnormally terminated. This is useful for low-end devices when their CPU consumption is very high.
```

**Review by omar-srbd — COMMENTED** [review]
Hello @seokwon-jung, please check the suggestions.

Thank you.

File: docs/platform/release-notes/tizen-8-0-m2.md (hunk 4 above)
Comment (safir-srbd) [reply docs/platform/release-notes/tizen-8-0-m2.md:326]: @seokwon-jung You missed this suggestion.

**Review by safir-srbd — COMMENTED** [review]
@seokwon-jung Thank you for implementing the suggested changes. However, it seems like you have missed a few suggestions. I have highlighted them. Please implement them.

**Review by omar-srbd — APPROVED** [review]
@seokwon-jung, thanks for incorporating all the changes.

@asm-foysal, I am approving this PR for merging.

**Pull request merged**